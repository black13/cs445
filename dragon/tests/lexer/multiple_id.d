boop
boop
boop boop boop      boop
hi
