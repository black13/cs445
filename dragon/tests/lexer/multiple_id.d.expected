ID(boop)
ID(boop)
ID(boop)
ID(boop)
ID(boop)
ID(boop)
ID(hi)
EOF
-- done dumping tokens --
